demos/storybook/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see http://editorconfig.org
```

Comment:
the editor config sneaked in

Reply:
This was an intentional add; this is actually the .editorconfig we have in our @pxblue/prettier-config package.  It finally has a use-case.